kent2cky/GeoSearch
# chore(finishing):Apply finishing touches

**What does the PR do?**

Apply finishing touches to the app

**Description of tasks to be completed**

- Add preloader functionality
- Add GeoSearch logo
- Update the about page
- Redesign result table layout
- Update the gulp file
- Update package.json and package-lock.json
- Add favicon.ico
    

**How should this be manually tested?**

- After cloning the repo, `cd` into it and run `npm install`
- After that, run the command `gulp` and it should be able to open up the page on your web browser
- After page has loaded type in the name of any place of choice and `click` on the `submit button` and the preloader should appear while information is being fetched.
- Clicking should be disable while the preloader is visible
- Preloader should be remove as soon as the results page is loaded
- Clicking on any of the blue map markers should have the preloader displayed while the image page is being fetched
- The preloader should disappear as soon as the image viewer is displayed
- The GeoSearch logo should appear at the top left corner of the page
- The about page should contain a clear description of what this app is about
- The search results should appear in a table layout 
- The favicon.ico should display in the browser tab that opens the GeoSearch page
- package.json and package-lock.json should contain latest dependencies for the GeoSearch app  

**Any background context you want to provide?**

Testers should have internet connection on.
Testers shoud contact the author if there are any issues running the app

**What are the relevant pivotal tracker stories?**

#3
